CHANGELOG.md

```diff
@@ -1,8 +1,19 @@
+## 2.12.3-dev.1
```

Comment:
let's actually do 2.13.0-dev.1 so that we are bumping to the next minor version

Reply:
and you'll want to specify the --since-tag of 2.12.0 when using the generate_changelog script (this script doesn't handle cherry pick releases very well, and both. 2.12.1 and 2.12.2 were cherry picks). then just make sure you remove the duplicate entries that are already included in 2.12.1 and 2.12.2